datasets/wegenbestand/dataset.json

```
@@ -0,0 +1,55 @@
+{
+  "id": "wegenbestand",
+  "type": "dataset",
+  "authorizationGrantor": "n.v.t.",
+  "theme": [
+    "verkeer"
+  ],
+  "homepage": "https://data.amsterdam.nl",
+  "owner": "Gemeente Amsterdam, bereikbaarheidsthermometer Amsterdam",
+  "dateModified": "2022-02-09T00:00:00+01:00",
+  "spatialDescription": "Gemeente Amsterdam",
+  "version": "1.0.0",
+  "default_version": "1.0.0",
+  "title": "Wegenbestand gemeente Amsterdam",
+  "language": "nl",
+  "dateCreated": "2022-02-09T00:00:00+01:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "hasBeginning": "2022-02-09T00:00:00+01:00",
+  "hasEnd": "",
+  "accrualPeriodicity": "dagelijks",
+  "publisher": "OIS",
```

```suggestion
  "publisher": "datapunt@amsterdam.nl",
```

Zie ook de output van GitHub CI.